src/main/java/org/gridsuite/explore/server/StudyService.java

```diff
@@ -65,14 +65,14 @@ public void setStudyServerBaseUri(String studyServerBaseUri) {
                 .log(ROOT_CATEGORY_REACTOR, Level.FINE);
     }
 
-    public Mono<Void> insertStudyWithCaseFile(UUID studyUuid, String studyName, String description, String userId, Boolean isPrivate, Mono<FilePart> caseFile) {
+    public Mono<Void> insertStudyWithCaseFile(UUID studyUuid, String userId, Boolean isPrivate, Mono<FilePart> caseFile) {
         return caseFile.flatMap(file -> {
             MultipartBodyBuilder multipartBodyBuilder = new MultipartBodyBuilder();
             multipartBodyBuilder.part("caseFile", file);
 
             String path = UriComponentsBuilder.fromPath(DELIMITER + STUDY_SERVER_API_VERSION +
-                            "/studies/{studyName}?description={description}&isPrivate={isPrivate}&studyUuid={studyUuid}")
-                    .buildAndExpand(studyName, description, isPrivate, studyUuid)
+                            "/studies?isPrivate={isPrivate}&studyUuid={studyUuid}")
```

Comment:
Maybe it would be better to use a URL like this:
"/studies/{studyUuid}?isPrivate={isPrivate}"
or do we want to allow creating a study without specifying its studyUuid ?

Reply:
After  thinking about it, we might want to create a study without specifying its uuid. I close this conversation.